apps/utils/lib/utils/json_rpc/response.ex

```diff
@@ -34,14 +43,30 @@ defmodule OMG.RPC.Web.Serializer.Response do
   end
 
   def sanitize(map_or_struct) when is_map(map_or_struct) do
-    map_or_struct
-    |> to_map()
-    |> Enum.map(fn {k, v} -> {k, sanitize(v)} end)
-    |> Map.new()
+    if Code.ensure_loaded?(Ecto) do
+      map_or_struct
+      |> to_map()
+      |> Enum.filter(fn {_k, v} -> Ecto.assoc_loaded?(v) end)
```

Comment:
if move if to 
```suggestion
      |> (if Code.ensure_loaded?(Ecto), do: &Enum.filter(&1, fn {_k, v} -> Ecto.assoc_loaded?(v) end), else: & &1)
```
no ned to cp code

Reply:
Ugh. I don't think this makes the code more readable. But willing to change if you both agree.